<!-- Launches nodes that are generally useful for manipulation. -->
<launch>
  <include file="$(find pr2_pick_manipulation)/launch/move_group.launch" />
  <node pkg="pr2_pick_manipulation" type="driver_service_node" name="driver_server" respawn="true" />
  <node pkg="pr2_pick_manipulation" type="gripper_service_node" name="gripper_server" respawn="true" />
  <node pkg="pr2_pick_manipulation" type="torso_service_node" name="torso_server" respawn="true" />
  <node pkg="pr2_pick_manipulation" type="tuck_arms_service_node" name="tuck_arms_server" respawn="true" />
  <node pkg="pr2_pick_manipulation" type="move_head_service_node" name="move_head_server" respawn="true" />
  <node pkg="pr2_tuck_arms_action" type="tuck_arms.py" name="tuck_arms_action_server" respawn="true" />
  <node pkg="pr2_pick_manipulation" type="moveit_service_node.py" name="moveit_service_node" respawn="true" />
</launch>
